packages/better-auth/tsconfig.json

```diff
@@ -14,7 +14,8 @@
 		"verbatimModuleSyntax": true,
 		"strict": true,
 		"noImplicitOverride": true,
-		"noFallthroughCasesInSwitch": true
+		"noFallthroughCasesInSwitch": true,
+		"experimentalDecorators": true
```

Comment:
This option enabled so I can define Mikro ORM entities with legacy decorators. If you want to keep it off, I can rewrite schema using [`EntitySchema`](https://mikro-orm.io/docs/entity-schema) and disable this option.